app/pages/lab/actions/project.js

```diff
@@ -10,7 +10,22 @@ var projectActions = {
       private: true
     }, projectData);
 
-    return projects.create(allProjectData).save();
+    return projects.create(allProjectData)
+      .save()
+      .then(function (projectResource) {
+        apiClient.post(projectResource._getURL('pages'), {
```

Comment:
Out of interest, why don't the other project pages get set up here?

Reply:
They get set up at https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/partials/project-page-editor.cjsx#L17. This page is different in that it'll be entirely autogenerated - the others have their resources created on first save.